central/main.go

```diff
@@ -245,6 +245,15 @@ func main() {
 	devmode.StartOnDevBuilds("central")
 
 	log.Infof("Running StackRox Version: %s", pkgVersion.GetMainVersion())
+	// TODO: ROX-12750 update with new list of replaced/deprecated resources
+	log.Warn("Next access resources will be replaced:\n" +
```

Comment:
```suggestion
	log.Warn("The following permission resources have been replaced:\n" +
```
Also please the same in the UI banner.